scrunch/datasets.py

```diff
@@ -323,6 +324,17 @@ def edit_user(self, user, edit):
             {user.url: {'permissions': {'edit': edit}}}
         )
 
+    @property
+    def order(self):
+        return self._order.get()
+
+    @order.setter
+    def order(self, _):
+        # Protect the `order` from external modifications.
+        raise TypeError(
+            'Unsupported operation on the Hierarchical Order property'
+        )
+
 
 class Path(object):
```

Comment:
Can we move the Path and Order classes outside of datasets.py, maybe to orders.py? To remove bloat from the single file